tests/topotests/multicast-pim-static-rp-topo1/multicast_pim_static_rp.json

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
Please fix the commit `root` is not an acceptable user name

Reply:
Done